src/Angor/Client/Components/MessageComponent.razor

```diff
@@ -0,0 +1,438 @@
+@using Angor.Client.Models
+@using Angor.Shared
+@using Angor.Shared.Services
+@using Angor.Client.Services
+@using Angor.Shared.Utilities
+@using Nostr.Client.Messages
```

Comment:
Should we allow the NuGet package objects to be referenced directly on the page?